crates/biome_console/src/markup.rs

```diff
@@ -182,6 +182,11 @@ impl Markup<'_> {
 pub struct MarkupBuf(pub Vec<MarkupNodeBuf>);
 
 impl MarkupBuf {
+    pub fn extend_with(&mut self, markup: Markup) {
```

Comment:
I'm sure there was a need for this, but:
- the PR doesn't mention why this was added
- the documentation doesn't explain when and how to use it